### ISSUE TITLE
Cannot find executable for CFBundle ... AWSAuthUI.bundle

### ISSUE THREAD

**Issue by desokroshan** [aws-amplify/aws-sdk-ios#1107]
Following the examples laid out (below) I'm getting an error when attempting to login to AWS Cognito.  This issue seems similar to #246 but I'm using Cognito, not Google.  For info, I'm using eu-west-1.  I've found in the past some of the samples seem to be fixed to us-east-1.

Example:
https://docs.aws.amazon.com/aws-mobile/latest/developerguide/getting-started.html
https://docs.aws.amazon.com/aws-mobile/latest/developerguide/add-aws-mobile-user-sign-in.html

Error:
```
2018-03-14 17:48:43:873 PropertyLife[32953:354651] didFinishLaunching:withOptions:resumeSessionWithCompletionHandler:
2018-03-14 17:48:43:882 PropertyLife[32953:354651] Resuming any previously signed-in session
2018-03-14 17:48:43.959737+0000 PropertyLife[32953:354651] Cannot find executable for CFBundle 0x7fa3489194e0 </Users/chris/Library/Developer/CoreSimulator/Devices/34C23101-6A34-464D-8361-7923659EE079/data/Containers/Bundle/Application/2D2F4AF1-5D29-4327-A3EF-5BA4F56A5764/PropertyLife.app/Frameworks/AWSAuthUI.framework/AWSAuthUI.bundle> (not loaded)
```

Originally opened as [sample Issue#251](https://github.com/awslabs/aws-sdk-ios-samples/issues/251) by @nomad3k

**Comment by rohandubal** [comment]
Hello @nomad3k 

We have identified a fix for this bug and will be fixed in a future release.

Thanks,
Rohan

**Comment by rohandubal** [comment]
Hello,

This issue is resolved in `2.7.4` SDK release. Please let us know if you continue to face issues.

Thanks,
Rohan
